### ISSUE TITLE
documentation may lead to accidentally include .git folders in their path, causing chaos 

### ISSUE THREAD

**Issue by smbaca** [sg-s/xolotl#584]
Please provide the following information:

## Operating System 
MacOS 12.4
## Output of "version" command in MATLAB
'9.12.0.1975300 (R2022a) Update 3'
## Output of "mex.getCompilerConfigurations('C++')" in MATLAB
             Name: 'Xcode Clang++'
     Manufacturer: 'Apple'
         Language: 'C++'
          Version: '13.1.6'
         Location: '/Applications/Xcode.app/Contents/Developer'
        ShortName: 'Clang++'
         Priority: 'A'
          Details: [1×1 mex.CompilerConfigurationDetails]
       LinkerName: '/usr/bin/xcrun -sdk macosx12.3 clang++'
    LinkerVersion: ''
           MexOpt: '/Users/smbaca/Library/Application Support/MathWorks/MATLAB/R2022a/mex_C++_maci64.xml'

# Bug reports

I installed xolotl using the git approach and setup all of the paths manually. All demos run fine when called individually. However, if I run xolotl.run_all_tests I get the following error and as far as I can tell all of the files are in the how-to folder:

All DONE! Found 1054 C++ files
Testing:
    "/Users/smbaca/Desktop/github_xolotl/xolotl/docs/how-to/change-existing-models.md"
    "/Users/smbaca/Desktop/github_xolotl/xolotl/docs/how-to/construct-models.md"
    "/Users/smbaca/Desktop/github_xolotl/xolotl/docs/how-to/copy-models.md"
    "/Users/smbaca/Desktop/github_xolotl/xolotl/docs/how-to/create-multi-compartment-models.md"
    "/Users/smbaca/Desktop/github_xolotl/xolotl/docs/how-to/database-search.md"
    "/Users/smbaca/Desktop/github_xolotl/xolotl/docs/how-to/discover-components.md"
    "/Users/smbaca/Desktop/github_xolotl/xolotl/docs/how-to/fit-parameters.md"
    "/Users/smbaca/Desktop/github_xolotl/xolotl/docs/how-to/inject-current.md"
    "/Users/smbaca/Desktop/github_xolotl/xolotl/docs/how-to/inspect-channels-and-mechanisms.md"
    "/Users/smbaca/Desktop/github_xolotl/xolotl/docs/how-to/install-configure.md"
    "/Users/smbaca/Desktop/github_xolotl/xolotl/docs/how-to/make-own-models.md"
    "/Users/smbaca/Desktop/github_xolotl/xolotl/docs/how-to/make-plots.md"
    "/Users/smbaca/Desktop/github_xolotl/xolotl/docs/how-to/manipulate.md"
    "/Users/smbaca/Desktop/github_xolotl/xolotl/docs/how-to/run-simulations-in-parallel.md"
    "/Users/smbaca/Desktop/github_xolotl/xolotl/docs/how-to/run-simulations.md"
    "/Users/smbaca/Desktop/github_xolotl/xolotl/docs/how-to/save-load-models.md"
    "/Users/smbaca/Desktop/github_xolotl/xolotl/docs/how-to/snapshots.md"
    "/Users/smbaca/Desktop/github_xolotl/xolotl/docs/how-to/specify-initial-conditions.md"
    "/Users/smbaca/Desktop/github_xolotl/xolotl/docs/how-to/stochastic-channels.md"
    "/Users/smbaca/Desktop/github_xolotl/xolotl/docs/how-to/voltage-clamp.md"
    "/Users/smbaca/Desktop/github_xolotl/xolotl/docs/how-to/write-mechanisms-that-self-assemble.md"

Error using assert
File does not exist

Error in filelib.extractCodeBlocks (line 14)
assert(exist(path_to_file,'file') == 2,'File does not exist')

Error in xolotl/testDocs (line 27)
	code_blocks = filelib.extractCodeBlocks(allfiles{i});

Error in xolotl.run_all_tests (line 52)
xolotl.testDocs()

**Comment by sg-s** [comment]
I got a different error in testDocs, which I fixed, please let me know if this error persists.

**Comment by smbaca** [comment]
Now I get this:

/Users/smbaca/Desktop/github_xolotl/xolotl/docs/how-to/change-existing-models.md
Error using xolotl/testDocs
Error while running /Users/smbaca/Desktop/github_xolotl/xolotl/docs/how-to/change-existing-models.md

Error in xolotl.run_all_tests (line 52)
xolotl.testDocs()

**Comment by sg-s** [comment]
@smbaca this is telling me that some code on [this page](https://xolotl.readthedocs.io/en/master/how-to/change-existing-models/) is failing on your computer. can you verify this independently?

**Comment by smbaca** [comment]
So everything runs on that page except this:

>> x = xolotl.examples.neurons.HodgkinHuxley;
x.HH.Kd.gbar = 300;
x.HH.Leak.gbar = 0.1;
x.HH.NaV.gbar = 1000;
2022-07-21 10:03:06.920 xcodebuild[54298:947603] Requested but did not find extension point with identifier Xcode.IDEKit.ExtensionSentinelHostApplications for extension Xcode.DebuggerFoundation.AppExtensionHosts.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
2022-07-21 10:03:06.921 xcodebuild[54298:947603] Requested but did not find extension point with identifier Xcode.IDEKit.ExtensionPointIdentifierToBundleIdentifier for extension Xcode.DebuggerFoundation.AppExtensionToBundleIdentifierMap.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
2022-07-21 10:03:07.519 xcodebuild[54303:947635] Requested but did not find extension point with identifier Xcode.IDEKit.ExtensionSentinelHostApplications for extension Xcode.DebuggerFoundation.AppExtensionHosts.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
2022-07-21 10:03:07.519 xcodebuild[54303:947635] Requested but did not find extension point with identifier Xcode.IDEKit.ExtensionPointIdentifierToBundleIdentifier for extension Xcode.DebuggerFoundation.AppExtensionToBundleIdentifierMap.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
2022-07-21 10:03:10.886 xcodebuild[54312:947704] Requested but did not find extension point with identifier Xcode.IDEKit.ExtensionSentinelHostApplications for extension Xcode.DebuggerFoundation.AppExtensionHosts.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
2022-07-21 10:03:10.886 xcodebuild[54312:947704] Requested but did not find extension point with identifier Xcode.IDEKit.ExtensionPointIdentifierToBundleIdentifier for extension Xcode.DebuggerFoundation.AppExtensionToBundleIdentifierMap.watchOS of plug-in com.apple.dt.IDEWatchSupportCore

**Comment by sg-s** [comment]
is that the full error message or is there more?

**Comment by smbaca** [comment]
Ha! 

Yeah, there is an OS update today and others have complained about this weird error with various solutions that work for some and not for all. I am going to go ahead and update the OS and make sure that the Xcode is also up to date. At some point the ability to compile worked fine but now I am getting FAILED for everything when I try xolotl.testConductances.

-smb

**Comment by smbaca** [comment]
That was the full message.

**Comment by sg-s** [comment]
i'm on macOS 12.4 too, and everything works for me. let me know how it works -- it's also possible that there are multiple copies of xolotl on your computer and that's confusing the compiler

**Comment by smbaca** [comment]
Yeah, I have only the one xolotl on the computer. Reinstalled a non-beta version of Xcode and am running 12.5.

xolotl.run_all_tests does everything fine until it gets to the testConductances and then they all fail.

/usr/bin/xcrun -sdk macosx12.3 clang++ this does not give any errors in terminal but I still think I have a weird mex_C++_maci64.xml file as I don’t see anyplace to change the LinkerName. That said, the LinkerName is correct so not sure what the issue is now. So close!

-smb

**Comment by sg-s** [comment]
oof, might want to stay away from the bleeding edge there. I'm on Xcode 13.4.1 -- what version are you on?

**Comment by smbaca** [comment]
Yes, I rolled back and think I cleaned everything out but still have the testconductances failing. I am using Xcode Version 13.4.1 (13F100). If someone can send their mex_C++_maci64.xml for comparison that might be helpful.

I have been in the various Mac XCode usergroups and a lot of people often have to reset in the terminal the default paths for various things anytime there is an Xcode upgrade. Maybe the roll back is not complete for all files or something. Who knows.

-smb

**Comment by sg-s** [comment]
have you seen this:

https://github.com/sg-s/matlab-catalina-compiler-fix

(you'll need to update the numbers for your version, but i wonder if this is the issue you're facing)

**Comment by smbaca** [comment]
Yes, I saw that and can give it a go. But I think the issue may be that xolotl is just not seeing the GitHub conductances folder…

I was going through the tutorials and noticed that if I search for prinz I get nothing even though I can go into my folder (which is also in the MATLAB path) and see that I have the prinz folder with 7 different .hpp files for the conductances.

xolotl.search('prinz')
------------------------------------------------------------------------------------
 Name                   Parent Class        Object Information
------------------------------------------------------------------------------------
------------------------------------------------------------------------------------

**Comment by sg-s** [comment]
there is something wrong with your path. before you mess around with xcode and compilers, let's fix that. 

can you send me your path? type `path` and copy over everything that is outside the MATLAB folder

**Comment by smbaca** [comment]
/Users/smbaca/Desktop/github_xolotl
	/Users/smbaca/Desktop/github_xolotl/cpplab
	/Users/smbaca/Desktop/github_xolotl/cpplab/.git
	/Users/smbaca/Desktop/github_xolotl/cpplab/.git/hooks
	/Users/smbaca/Desktop/github_xolotl/cpplab/.git/info
	/Users/smbaca/Desktop/github_xolotl/cpplab/.git/logs
	/Users/smbaca/Desktop/github_xolotl/cpplab/.git/logs/refs
	/Users/smbaca/Desktop/github_xolotl/cpplab/.git/logs/refs/heads
	/Users/smbaca/Desktop/github_xolotl/cpplab/.git/logs/refs/remotes
	/Users/smbaca/Desktop/github_xolotl/cpplab/.git/logs/refs/remotes/origin
	/Users/smbaca/Desktop/github_xolotl/cpplab/.git/objects
	/Users/smbaca/Desktop/github_xolotl/cpplab/.git/objects/info
	/Users/smbaca/Desktop/github_xolotl/cpplab/.git/objects/pack
	/Users/smbaca/Desktop/github_xolotl/cpplab/.git/refs
	/Users/smbaca/Desktop/github_xolotl/cpplab/.git/refs/heads
	/Users/smbaca/Desktop/github_xolotl/cpplab/.git/refs/remotes
	/Users/smbaca/Desktop/github_xolotl/cpplab/.git/refs/remotes/origin
	/Users/smbaca/Desktop/github_xolotl/cpplab/.git/refs/tags
	/Users/smbaca/Desktop/github_xolotl/cpplab/cache
	/Users/smbaca/Desktop/github_xolotl/cpplab/docs
	/Users/smbaca/Desktop/github_xolotl/cpplab/docs/reference
	/Users/smbaca/Desktop/github_xolotl/puppeteer
	/Users/smbaca/Desktop/github_xolotl/puppeteer/.git
	/Users/smbaca/Desktop/github_xolotl/puppeteer/.git/hooks
	/Users/smbaca/Desktop/github_xolotl/puppeteer/.git/info
	/Users/smbaca/Desktop/github_xolotl/puppeteer/.git/logs
	/Users/smbaca/Desktop/github_xolotl/puppeteer/.git/logs/refs
	/Users/smbaca/Desktop/github_xolotl/puppeteer/.git/logs/refs/heads
	/Users/smbaca/Desktop/github_xolotl/puppeteer/.git/logs/refs/remotes
	/Users/smbaca/Desktop/github_xolotl/puppeteer/.git/logs/refs/remotes/origin
	/Users/smbaca/Desktop/github_xolotl/puppeteer/.git/objects
	/Users/smbaca/Desktop/github_xolotl/puppeteer/.git/objects/info
	/Users/smbaca/Desktop/github_xolotl/puppeteer/.git/objects/pack
	/Users/smbaca/Desktop/github_xolotl/puppeteer/.git/refs
	/Users/smbaca/Desktop/github_xolotl/puppeteer/.git/refs/heads
	/Users/smbaca/Desktop/github_xolotl/puppeteer/.git/refs/remotes
	/Users/smbaca/Desktop/github_xolotl/puppeteer/.git/refs/remotes/origin
	/Users/smbaca/Desktop/github_xolotl/puppeteer/.git/refs/tags
	/Users/smbaca/Desktop/github_xolotl/srinivas.gs_mtools
	/Users/smbaca/Desktop/github_xolotl/srinivas.gs_mtools/.git
	/Users/smbaca/Desktop/github_xolotl/srinivas.gs_mtools/.git/hooks
	/Users/smbaca/Desktop/github_xolotl/srinivas.gs_mtools/.git/info
	/Users/smbaca/Desktop/github_xolotl/srinivas.gs_mtools/.git/logs
	/Users/smbaca/Desktop/github_xolotl/srinivas.gs_mtools/.git/logs/refs
	/Users/smbaca/Desktop/github_xolotl/srinivas.gs_mtools/.git/logs/refs/heads
	/Users/smbaca/Desktop/github_xolotl/srinivas.gs_mtools/.git/logs/refs/remotes
	/Users/smbaca/Desktop/github_xolotl/srinivas.gs_mtools/.git/logs/refs/remotes/origin
	/Users/smbaca/Desktop/github_xolotl/srinivas.gs_mtools/.git/objects
	/Users/smbaca/Desktop/github_xolotl/srinivas.gs_mtools/.git/objects/info
	/Users/smbaca/Desktop/github_xolotl/srinivas.gs_mtools/.git/objects/pack
	/Users/smbaca/Desktop/github_xolotl/srinivas.gs_mtools/.git/refs
	/Users/smbaca/Desktop/github_xolotl/srinivas.gs_mtools/.git/refs/heads
	/Users/smbaca/Desktop/github_xolotl/srinivas.gs_mtools/.git/refs/remotes
	/Users/smbaca/Desktop/github_xolotl/srinivas.gs_mtools/.git/refs/remotes/origin
	/Users/smbaca/Desktop/github_xolotl/srinivas.gs_mtools/.git/refs/tags
	/Users/smbaca/Desktop/github_xolotl/srinivas.gs_mtools/docs
	/Users/smbaca/Desktop/github_xolotl/srinivas.gs_mtools/docs/veclib
	/Users/smbaca/Desktop/github_xolotl/srinivas.gs_mtools/licenses
	/Users/smbaca/Desktop/github_xolotl/srinivas.gs_mtools/src
	/Users/smbaca/Desktop/github_xolotl/srinivas.gs_mtools/src/beta
	/Users/smbaca/Desktop/github_xolotl/srinivas.gs_mtools/src/deprecated
	/Users/smbaca/Desktop/github_xolotl/srinivas.gs_mtools/src/t_sne
	/Users/smbaca/Desktop/github_xolotl/xolotl
	/Users/smbaca/Desktop/github_xolotl/xolotl/.git
	/Users/smbaca/Desktop/github_xolotl/xolotl/.git/hooks
	/Users/smbaca/Desktop/github_xolotl/xolotl/.git/info
	/Users/smbaca/Desktop/github_xolotl/xolotl/.git/logs
	/Users/smbaca/Desktop/github_xolotl/xolotl/.git/logs/refs
	/Users/smbaca/Desktop/github_xolotl/xolotl/.git/logs/refs/heads
	/Users/smbaca/Desktop/github_xolotl/xolotl/.git/logs/refs/remotes
	/Users/smbaca/Desktop/github_xolotl/xolotl/.git/logs/refs/remotes/origin
	/Users/smbaca/Desktop/github_xolotl/xolotl/.git/objects
	/Users/smbaca/Desktop/github_xolotl/xolotl/.git/objects/info
	/Users/smbaca/Desktop/github_xolotl/xolotl/.git/objects/pack
	/Users/smbaca/Desktop/github_xolotl/xolotl/.git/refs
	/Users/smbaca/Desktop/github_xolotl/xolotl/.git/refs/heads
	/Users/smbaca/Desktop/github_xolotl/xolotl/.git/refs/remotes
	/Users/smbaca/Desktop/github_xolotl/xolotl/.git/refs/remotes/origin
	/Users/smbaca/Desktop/github_xolotl/xolotl/.git/refs/tags
	/Users/smbaca/Desktop/github_xolotl/xolotl/.github
	/Users/smbaca/Desktop/github_xolotl/xolotl/build
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/amarillo
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/bronk
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/brookings
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/caplan
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/chow
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/clay00
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/custom
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/destexhe
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/destexhe98
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/dethier
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/generic
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/giovannini
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/goldman
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/golowasch
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/hardie-laughlin
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/heras
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/hill
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/hodgkin-huxley
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/jochems
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/kispersky
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/lin
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/liu
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/liu-temperature
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/morris-lecar
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/nadim98
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/nadim98/int1
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/nadim98/lg
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/nadim98/mcn1
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/nadim99
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/odowd-aldrich
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/prinz
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/prinz-temperature
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/rodriguez
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/rodriguez/Int1
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/rodriguez/LG
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/sharp
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/sharp96
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/soplata
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/soplata/legacy
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/soplata/legacy/reticular
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/soplata/legacy/thalamocortical
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/soto-trevino01
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/soto-trevino01/ABPD
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/soto-trevino01/LP
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/soto-trevino01/PY
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/soto-trevino05
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/soto-trevino05/AB
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/soto-trevino05/PD
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/swensen
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/templates
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/traub
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/turrigiano
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/conductances/wicher
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/mechanisms
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/mechanisms/buchholtz
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/mechanisms/buchholtz-temperature
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/mechanisms/destexhe
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/mechanisms/goldwyn
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/mechanisms/gorur-shandilya
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/mechanisms/liu
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/mechanisms/oleary
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/mechanisms/prinz
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/mechanisms/prinz-temperature
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/mechanisms/soto-trevino
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/mechanisms/traub
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/synapses
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/synapses/borgers
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/synapses/cymbalyuk
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/synapses/dayan-abbott
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/synapses/generic
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/synapses/gutierrez
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/synapses/nadim
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/synapses/nadim98
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/synapses/prinz
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/synapses/prinz-temperature
	/Users/smbaca/Desktop/github_xolotl/xolotl/c++/synapses/soplata
	/Users/smbaca/Desktop/github_xolotl/xolotl/docs
	/Users/smbaca/Desktop/github_xolotl/xolotl/docs/explanation
	/Users/smbaca/Desktop/github_xolotl/xolotl/docs/how-to
	/Users/smbaca/Desktop/github_xolotl/xolotl/docs/images
	/Users/smbaca/Desktop/github_xolotl/xolotl/docs/meta
	/Users/smbaca/Desktop/github_xolotl/xolotl/docs/performance
	/Users/smbaca/Desktop/github_xolotl/xolotl/docs/reference
	/Users/smbaca/Desktop/github_xolotl/xolotl/docs/reference/c++
	/Users/smbaca/Desktop/github_xolotl/xolotl/docs/reference/matlab
	/Users/smbaca/Desktop/github_xolotl/xolotl/docs/tutorials
	/Users/smbaca/Desktop/github_xolotl/xolotl/examples
	/Users/smbaca/Desktop/github_xolotl/xolotl/examples/xgrid
	/Users/smbaca/Desktop/github_xolotl/xolotl/experimental
	/Users/smbaca/Desktop/github_xolotl/xolotl/experimental/proof-of-concept
	/Users/smbaca/Desktop/github_xolotl/xolotl/git-hooks
	/Users/smbaca/Desktop/github_xolotl/xolotl/optimization

**Comment by sg-s** [comment]
there's your problem  right there -- you have .git folders on your path. this means there are multiple versions of code on your path -- a very bad idea.

**Comment by smbaca** [comment]
Unfortunately, now I even have more bizarre errors after removing those folders. I have another Mac so will try to start fresh from there as I feel there are residual folders or preferences that are mucking up things.

-smb

**Comment by smbaca** [comment]
Yeah, this is really upside-down world at this point. I feel like there are some low level preferences that aren’t getting wiped. At this point I have reinstalled both MATLAB and Xcode and there are still weird errors relating to the path. Going to try on a different Mac because at this point it seems like I need to do a deep clean and or system re-install.

-smb

**Comment by smbaca** [comment]
Ok, I see now that some all the regular folders need to be in the path. When I did that it looks like things are now up and running. There are a couple of conductances that failed but the rest are not showing OK.

Just in time for my vacation but at least I know I can work on the planned models when I return.

Thanks for all of the fast feedback. I am sure I will have more questions down the line.

-Serapio

BELOW are the only Warnings or errors I received when I ran xolotl.run_all_tests

Warning: Saving object of class 'parallel.FevalFuture' is not supported. Avoid storing objects of this
class in MAT files or sending them between MATLAB Sessions. 
Starting worker #6

demo_manipulate
Warning: File 'test.m' not found. 
> In xolotl.testDocs (line 44)
In xolotl.run_all_tests (line 52)

**Comment by sg-s** [comment]
excellent, glad something is working!
